openfisca_france/model/prestations/prestations_familiales/paje.py

```diff
@@ -236,6 +236,13 @@ def function(self, simulation, period):
                                 partiel2 * P.paje.clca.avecab_tx_partiel2))
         return period, paje_clca
 
+reference_input_variable(
+   name ='paje_prepare',
+   column = FloatCol,
+   entity_class = Familles,
+   label = u"Prestation partagée d’éducation de l’enfant",
```

Comment:
Please introduce the acronym PreParE so people can get the origin of the variable name.